modules/entrezdirect/esummary/main.nf

```diff
@@ -0,0 +1,35 @@
+process ENTREZDIRECT_ESUMMARY {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::entrez-direct=13.9" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/entrez-direct:13.9--pl526h375a9b1_1':
+        'quay.io/biocontainers/entrez-direct:13.9--pl526h375a9b1_1' }"
```

Comment:
1. Any reason not to use the latest version (16.2)?

e.g. quay.io/biocontainers/entrez-direct:16.2--he881be0_1 for docker

I _can_ replicate the conda error you're getting on the CI tests with conda, but updating the module to 16.2 fixes this issue.

2. Alternatively, no tested but I also see there was a third build of 13.9 (build `_2`), you could try that instead:
[entrez-direct:13.9--pl5262he881be0_2](https://depot.galaxyproject.org/singularity/entrez-direct%3A13.9--pl5262he881be0_2)               25-Mar-2021 08:46            78024704
[entrez-direct:13.9--pl526h375a9b1_0](https://depot.galaxyproject.org/singularity/entrez-direct%3A13.9--pl526h375a9b1_0)                21-Sep-2020 04:49            48795648
[entrez-direct:13.9--pl526h375a9b1_1](https://depot.galaxyproject.org/singularity/entrez-direct%3A13.9--pl526h375a9b1_1)                03-Feb-2021 10:33            77901824

maybe that fixes the crash which is the following:

```
Can't locate Net/SSLeay.pm in @INC (you may need to install the Net::SSLeay module) (@INC contains: /home/jfellows/.conda/cache/env-aa3fd40ca4355b903cbfaa499a859056/bin/aux/lib/perl5 /home/jfellows/.conda/cache/env-aa3fd40ca4355b903cbfaa499a859056/lib/site_perl/5.26.2/x86_64-linux-thread-multi /home/jfellows/.conda/cache/env-aa3fd40ca4355b903cbfaa499a859056/lib/site_perl/5.26.2 /home/jfellows/.conda/cache/env-aa3fd40ca4355b903cbfaa499a859056/lib/5.26.2/x86_64-linux-thread-multi /home/jfellows/.conda/cache/env-aa3fd40ca4355b903cbfaa499a859056/lib/5.26.2 .) at /home/jfellows/.conda/cache/env-aa3fd40ca4355b903cbfaa499a859056/lib/site_perl/5.26.2/IO/Socket/SSL.pm line 19.
  BEGIN failed--compilation aborted at /home/jfellows/.conda/cache/env-aa3fd40ca4355b903cbfaa499a859056/lib/site_perl/5.26.2/IO/Socket/SSL.pm line 19.
  Compilation failed in require at /home/jfellows/.conda/cache/env-aa3fd40ca4355b903cbfaa499a859056/bin/edirect.pl line 70.
  BEGIN failed--compilation aborted at /home/jfellows/.conda/cache/env-aa3fd40ca4355b903cbfaa499a859056/bin/edirect.pl line 70.
```

Reply:
Yest, there is an error that seems to be related to a dependency in the container.

Reply:
Strange, I just tested both this module and xtract with 16.2 and both work fine for conda/docker/singularity. As the tests are passing on this module now I assume maybe there was a new build that fixed it so we can preferably stick with this :+1: